### PR TITLE
add example for generating pdf

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

File: modules/pdf_generation/public/views/pages/pdf_generation.liquid [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+slug: pdf-generation
+metadata:
+  description: Example of generating PDF from html
+---
+{% comment %}
+  This page needs some sisters/borthers, showcasing different formats, etc.
```
Comment (Developer-DNM) [inline modules/pdf_generation/public/views/pages/pdf_generation.liquid]: brothers

**Comment by pavelloz** [comment]
tests? ;>

File: modules/pdf_generation/public/views/pages/pdf_generation.liquid (hunk 1 above)
Comment (Slashek) [reply modules/pdf_generation/public/views/pages/pdf_generation.liquid]: ;)

**Comment by Slashek** [comment]
was hoping you could help with those .. :)

**Comment by pavelloz** [comment]
cc @jnorbert

**Pull request merged**